bw2io/extractors/simapro_csv.py

```diff
@@ -359,7 +560,8 @@ def parse_calculated_parameter(cls, line, pm):
 
     @classmethod
     def parse_input_parameter(cls, line):
-        """Parse line in `Input parameters` section.
+        """
+        Parse input parameters section of a SimaPro file.
 
         0. name
```

Comment:
@cmutel, can you check if this needs to be removed also?

Reply:
Sorry, I guess I missed some connection - not sure why this would be removed specifically or in general, it is important. I wish we could remove it, but life is not that kind.

Reply:
...just seemed like the list was incomplete. Was GitHub hiding un-diffed lines.